tracing-core/src/metadata.rs

```diff
@@ -185,6 +192,12 @@ impl<'a> Metadata<'a> {
         self.line
     }
 
+    /// Returns the column number in the source code file where the span
+    /// occurred, or `None` if the column number is unknown.
```

Comment:
nit: this `Metadata` could describe a span _or_ an event, so this might be nicer as
```suggestion
    /// Returns the column number in the source code file where this
    /// `Metadata` originated from, or `None` if the column number is unknown.
```